src/Gtk/Avalonia.Gtk3/Gtk3Platform.cs

```diff
@@ -148,8 +148,8 @@ public void Signal(DispatcherPriority prio)
 
     public class Gtk3PlatformOptions
     {
-        public bool? UseDeferredRendering { get; set; }
-        public bool? UseGpuAcceleration { get; set; }
+        public bool? UseDeferredRendering { get; set; } = true;
```

Comment:
Default value should be set here: https://github.com/AvaloniaUI/Avalonia/blob/7add17a39b4e72da26c5126527ee52c48e94b3c6/src/Gtk/Avalonia.Gtk3/Gtk3Platform.cs#L52
Property was made nullable for a reason